test/e2e/mesos.go

```diff
@@ -106,10 +106,10 @@ var _ = Describe("Mesos", func() {
 		expectNoError(err)
 
 		nodeClient := framework.Client.Nodes()
-		role1 := labels.SelectorFromSet(map[string]string{
-			"k8s.mesosphere.io/attribute-role": "role1",
+		rack2 := labels.SelectorFromSet(map[string]string{
```

Comment:
Add a comment like "// schedule onto node with rack=2 which is the same that has the "public" resources"